category.php

```diff
@@ -13,25 +13,26 @@
 
 get_header(); ?>
 
-	<section id="primary" class="site-content">
-		<div id="content" role="main">
+    <section id="primary" class="site-content">
+        <div id="content" role="main">
 
 		<?php if ( have_posts() ) : ?>
-			<header class="archive-header">
+            <header class="archive-header">
 				<h1 class="archive-title"><?php printf( __( 'Category Archives: %s', 'twentytwelve' ), '<span>' . single_cat_title( '', false ) . '</span>' ); ?></h1>
 
 			<?php if ( category_description() ) : // Show an optional category description ?>
 				<div class="archive-meta"><?php echo category_description(); ?></div>
 			<?php endif; ?>
-			</header><!-- .archive-header -->
+            </header><!-- .archive-header -->
 
 			<?php
 			/* Start the Loop */
 			while ( have_posts() ) : the_post();
 
-				/* Include the post format-specific template for the content. If you want to
-				 * this in a child theme then include a file called called content-___.php
-				 * (where ___ is the post format) and that will be used instead.
+				/*
```

Comment:
Yeah, I'm seeing the same sort of weirdness in the hours library, for example: https://github.com/MITLibraries/MITlibraries-parent/blob/4ec6389289534d86b9059af5939f96915eb9b42b/lib/hours.php#L875

This probably rules out this tool as it is currently configured. I don't want to have to crawl through this large a diff looking for unintended consequences.